static/docs/commands-reference/run.md

```diff
@@ -146,6 +146,12 @@ pipeline.
 
   > The `dvc status` command will mention that the file is `not in cache`.
 
+- `--outs-persist` - the same as `-o` except that outputs will not be removed
```

Comment:
please, remove this - there is PR for this - https://github.com/iterative/dvc.org/pull/217/files

it's not being merged for a reason. These options are still semi-official.

Reply:
btw, that PR is a good example how the option should be described.

Reply:
@shcheklein remove both `--outs-persist` and `--outs-persist-no-cache` option?

Reply:
> btw, that PR is a good example how the option should be described.

yes, examples are mentioned also. will try to introduce new information in such a way from next time.